crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/validation/service/MeansAssessmentValidationService.java

```diff
@@ -88,4 +92,12 @@ public boolean isRepOrderReserved(final MeansAssessmentRequestDTO meansAssessmen
         }
         return false;
     }
+
+    public boolean isAssessmentModifiedByAnotherUser(final MeansAssessmentRequestDTO meansAssessmentRequest) {
+        FinancialAssessmentDTO financialAssessmentDTO = maatCourtDataService.getFinancialAssessment(meansAssessmentRequest.getFinancialAssessmentId(),
```

Comment:
Potential for a NullPointerException here if no financial assessment is returned.

Reply:
Added null check